lib/label/add-label.js

```diff
@@ -6,7 +6,10 @@ module.exports = addLabel;
 
 function addLabel(root, node, location) {
   var label = node.label;
-  var labelSvg = root.append("g");
+  var labelSvg = root.select("g");
```

Comment:
The functions addHtmlLabel, addSvgLabel and addTextLabel will create new nodes each time they are called within the same "g". You can see it visually if you add a Html label and call render twice.

I suggest that you should:

 - select the "g" in the beginning of this function and store it in a variable like _oldLabelSvg_
 - call append("g") always as it was before
 - do not do checking the existence of any node in any of the addXXXLabel functions as it will not be necessary anymore
 - remove oldLabelSvg in the end of the function if it exists to avoid multiple "g" nodes in the svg